src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXOverrideMismatch/PXOverrideMethodSymbolComparer.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using System.Linq;
+using Acuminator.Utilities.Common;
+
+namespace Acuminator.Analyzers.StaticAnalysis.PXOverrideMismatch
+{
+	public class PXOverrideMethodSymbolComparer : IEqualityComparer<IMethodSymbol>
+	{
+		public static PXOverrideMethodSymbolComparer Instance = new PXOverrideMethodSymbolComparer();
+
+		public bool Equals(IMethodSymbol src, IMethodSymbol ext)
+		{
+			var srcParams = src.Parameters.Select(p => p.Type).ToImmutableArray();
```

Comment:
We have extra requirements for the performance since this code is running constantly in the IDE on the developer machine. This method can be made more performant:
You always do LINQ transformation and then always buffer it into the Immutable array even if there is no need to do it.  You can check parameters length before any need to use any parameter type. Moreover, this conversion is not needed at all. No need to allocate two more collections when you can work directly with parameters collections.